`make generate-ami` no longer works

**What happened?**
Generating AMIs for use with the static resolver no longer works due to a regression.

**How to reproduce it?**
```shell
make generate-ami
```

**Versions**
Please paste in the output of these commands:
```
$ eksctl version
[ℹ]  version.Info{BuiltAt:"", GitCommit:"", GitTag:"0.4.1"}

```

**Logs**
```shell
$ make generate-ami
time go generate ./pkg/ami
# command-line-arguments
./static_resolver_ami_generate.go:41:22: undefined: ami.ImageFamilyToAccountID
pkg/ami/static_resolver.go:8: running "go": exit status 2
Command exited with non-zero status 1
0.89user 0.19system 0:00.71elapsed 152%CPU (0avgtext+0avgdata 70920maxresident)k
107448inputs+6088outputs (381major+39283minor)pagefaults 0swaps
Makefile:162: recipe for target 'generate-ami' failed
make: *** [generate-ami] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -156,7 +156,7 @@ $(DEEP_COPY_HELPER): $(DEEP_COPY_DEPS) .license-header ## Generate Kubernetes A
 	  || (cat codegenheader.txt ; cat $(DEEP_COPY_HELPER); exit 1)
 
 # static_resolver_ami.go doesn't only depend on files (it should be refreshed whenever a release is made in AWS)
-# so we need to forcicly generate it
+# so we need to forcibly generate it
 .PHONY: generate-ami
 generate-ami: ## Generate the list of AMIs for use with static resolver. Queries AWS.
 	time go generate ./pkg/ami

--- a/pkg/ami/auto_resolver.go
+++ b/pkg/ami/auto_resolver.go
@@ -43,7 +43,7 @@ var ImageSearchPatterns = map[string]map[string]map[int]string{
 }
 
 // OwnerAccountID returns the AWS account ID that owns worker AMI.
-func OwnerAccountID(imageFamily string, region string) (string, error) {
+func OwnerAccountID(imageFamily, region string) (string, error) {
 	switch imageFamily {
 	case ImageFamilyUbuntu1804:
 		return "099720109477", nil

--- a/pkg/ami/static_resolver_ami_generate.go
+++ b/pkg/ami/static_resolver_ami_generate.go
@@ -38,7 +38,10 @@ func main() {
 				classImages := Dict{}
 				for _, region := range api.SupportedRegions() {
 					namePattern := ami.ImageSearchPatterns[version][family][class]
-					ownerAccount := ami.ImageFamilyToAccountID[family]
+					ownerAccount, err := ami.OwnerAccountID(family, region)
+					if err != nil {
+						log.Fatal(err)
+					}
 					log.Printf("looking up images matching %q in %q", namePattern, region)
 					id, err := ami.FindImage(client[region], ownerAccount, namePattern)
 					if err != nil {

